Perlang.Tests.Integration/Operator/AdditionAssignment.cs

```diff
@@ -111,7 +111,7 @@ public void addition_assignment_to_string_throws_expected_exception()
             var exception = result.Errors.First();
 
             Assert.Single(result.Errors);
-            Assert.Matches("Invalid arguments to operator PLUS_EQUAL specified", exception.Message);
+            Assert.Equal("Invalid arguments to += operator specified", exception.Message);
```

Comment:
This (and all related changes) is a drive-by change I did. The previous approach, of just writing out the token name in `UPPER_SNAKE` looked a bit ugly; the new approach aims for a more user-friendly error message.